docs/blog/write-the-docs-2020-stats.rst

```diff
@@ -0,0 +1,224 @@
+.. post:: Jan 12, 2021
+   :tags: stats, year in review, newsletter
+   :author: Eric Holscher
+
+Write the Docs 2020 Stats and Community Update
+==============================================
+
+A new year comes with many expectations.
```

Comment:
Should we indicate that this is instead of the January newsletter? Also, maybe specify that it's coming from Eric, as Beth usually greets in person..